tests/unit/rest/user_test.py

```diff
@@ -27,16 +27,17 @@ def test_user_authenticated(self):
             {
                 "type": "users",
                 "attributes": {
-                    "email": "",
-                    "first_name": "",
+                    "email": "codalab@codalab.com",
```

Comment:
😄 

```suggestion
                    "email": "codalab@codalab.org",
```

Reply:
We should change the email here. Since the standard answer is generated as `codalab@codalab.com`

Reply:
Oh, ok. I think we should actually change this back to what it was before, since the test is failing on CI:

```
ssertionError: {'typ[24 chars]': {'disk_quota': 107374000000, 'last_name': '[188 chars] '0'} != {'typ[24 chars]': {'email': 'codalab@codalab.com', 'first_nam[229 chars] '0'}
- {'attributes': {'affiliation': '',
+ {'attributes': {'affiliation': 'Stanford University',
?                                 +++++++++++++++++++

                  'avatar_id': None,
                  'disk_quota': 107374000000,
-                 'email': '',
+                 'email': 'codalab@codalab.com',
-                 'first_name': '',
+                 'first_name': 'Coda',
?                                ++++

-                 'last_name': '',
+                 'last_name': 'Lab',
?                               +++

                  'notifications': 2,
                  'parallel_run_quota': 100,
-                 'time_quota': 3153600000,
?                               ^^^^ ^^

+                 'time_quota': 864000,
?                               ^ ^

                  'url': None,
                  'user_name': 'codalab'},
   'id': '0',
   'type': 'users'}
-------------------- >> begin captured logging << --------------------
oauthlib.oauth2.rfc6749.endpoints.resource: DEBUG: Dispatching token_type *** to <oauthlib.oauth2.rfc6749.tokens.BearerToken object at 0x7f4e0dcbb6d0>.
codalab.server.oauth2_provider: DEBUG: Validate bearer token None
codalab.server.oauth2_provider: DEBUG: *** not found.
--------------------- >> end captured logging << ---------------------

======================================================================
FAIL: A MakeBundle with no dependencies should be made.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/opt/codalab-worksheets/tests/unit/server/bundle_manager/make_bundles_test.py", line 52, in test_bundle_no_dependencies
    self.make_bundles_and_wait()
  File "/opt/codalab-worksheets/tests/unit/server/bundle_manager/make_bundles_test.py", line 27, in make_bundles_and_wait
    self.assertTrue(self.bundle_manager._is_making_bundles())
AssertionError: False is not true
-------------------- >> begin captured logging << --------------------
codalab.server.bundle_manager: INFO: Making bundle 0xeebdddbfcdc742feb0326f921fe89cdd
codalab.server.bundle_manager: INFO: Finished making bundle 0xeebdddbfcdc742feb0326f921fe89cdd
--------------------- >> end captured logging << ---------------------

----------------------------------------------------------------------
Ran 130 tests in 11.764s

FAILED (SKIP=1, errors=1, failures=2)
 (exit code 1, expected 0 BAD)
```

Reply:
I see.

Reply:
Updated